src/app/zap-templates/templates/app/helper.js

```diff
@@ -186,6 +187,133 @@ function asChipUnderlyingType(label, type)
   }
 }
 
+//  Endpoint-config specific helpers
+// these helpers are a Hot fix for the "GENERATED_FUNCTIONS" problem
+// They should be removed or replace once issue #4369 is resolved
+// These helpers only works within the endpoint_config iterator
+
+// List of all cluster with generated functions
+var endpointClusterWithInit = [ 'Identify', 'Groups', 'Scenes', 'On/off', 'Level Control', 'Color Control', 'IAS Zone' ];
+var endpointClusterWithAttributeChanged = [ 'Identify', 'Door Lock' ];
+var endpointClusterWithPreAttribute     = [ 'IAS Zone' ];
+var endpointClusterWithMessageSent      = [ 'IAS Zone' ];
+
```

Comment:
Looks good but won't work unless `clusterGeneratedFunctions` includes all possible cluster. Otherwise you'll get cannot access property length of undefined in the `if (clusterGeneratedFunctions[clusterName].length)`. I would keep it as it is since it is a hot fix and this will be improve later on.